onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -269,10 +269,32 @@ def aten_angle(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_any(self: TensorType) -> TensorType:
+@torch_op("aten::any", trace_only=True)
+def aten_any(self: TTensor, dim: Optional[int] = None, keepdim: bool = True) -> BOOL:
```

Comment:
Is `dim` meant to be an attribute or input? I guess it is meant to be an attribute?

Reply:
yes, it is an attribute from the test case.